### PR TITLE
add an option to add a hash in generated font file name (fixes #163)

### LINKED ISSUE #163: Add a file version hash
Hi,

It would be great to have the option to have a version query string added to the fontface declaration to help with cache busting.

### PR DESCRIPTION
**Pull request by brunoroux**
How to test

```
npm run build && node dist/cli.js './src/__tests__/fixtures/svg-icons/*.svg' -d demo -t css --normalize --center-horizontally --add-hash-in-font-url
npm run build && node dist/cli.js './src/__tests__/fixtures/svg-icons/*.svg' -d demo -t html --normalize --center-horizontally --add-hash-in-font-url
npm run build && node dist/cli.js './src/__tests__/fixtures/svg-icons/*.svg' -d demo -t scss --normalize --center-horizontally --add-hash-in-font-url
npm run build && node dist/cli.js './src/__tests__/fixtures/svg-icons/*.svg' -d demo -t styl --normalize --center-horizontally --add-hash-in-font-url
```

### CODE REVIEW

File: src/__tests__/standalone-test.js [hunk 1]
```diff
@@ -426,4 +426,16 @@ describe("standalone", () => {
       return result;
     });
   });
+
+  it("should add fon hash in result", () => {
```
Comment (zoolette) [inline src/__tests__/standalone-test.js]: Typo : font

**Comment by brunoroux** [comment]
Could you please explain a bit more what do you mean ?
I'm not sure I'm getting it. Do you mean adding the hash only in the url ?

**Comment by brunoroux** [comment]
is this more what you expected ?

**Comment by evilebottnawi** [comment]
Yes, you can implement own template and add hash to url, i can add example

**Comment by Connum** [comment]
Any progress? I think this is an essential feature! If it's solvable with a custom template - how would you get the file hash inside the template?

File: src/__tests__/standalone.test.js [hunk 2]
```diff
@@ -366,4 +366,16 @@ describe("standalone", () => {
...
     expect(result.glyphsData.length > 0).toBe(true);
   });
+
+  it("should export `hash` in `result`", () => {
+    expect.assertions(1);
+
+    return standalone({
+      files: `${fixturesGlob}/svg-icons/**/*`
+    }).then(result => {
+      expect(result.hash).toBe("ead2b6f69fc603bf1cbd00bf9f8a8a33");
```
Comment (jimmyandrade) [inline src/__tests__/standalone.test.js:376]: Is this hash mocked-up?

**Comment by jimmyandrade** [comment]
> Any progress? I think this is an essential feature! If it's solvable with a custom template - how would you get the file hash inside the template?

@Connum sorry for the delay in returning. We just made some necessary adjustments and merged it with the main branch.

We have not yet released a new version with support for this feature, I will check this with @evilebottnawi.

**Pull request merged**